### DISCUSSION TITLE
2 PCE games not work well

### DISCUSSION THREAD

**Discussion by alexgame8818** [darthcloud/BlueRetro#466] [Q&A]
That's very interesting to build these Blueretros, thank you:)

FM 1.7.3
About PCE games I found Jackie Chan and bomberman 93&94 not work well.
In JackieChan，NONE option will occur PAUSE sometimes. 
If using Slot1 option（5players）it will happen very frequently.(Other games work well.)

In bomberman93 or 94,the 5th player will be the killer:)
When 5th player set a bomb, other four players will set a bomb too.
It seemed 5th player pressed a key for Everyone.

I used PS4,PS5,WIIUPRO,SWITCH PRO,SWITCH JOYCON to test these three games.
No matter which controller I use，the results are the same.

Because other games work very well，I dont know if there's problem on my PCB.

Even these problems happen, I am still very happy to play other games.Thank you again:)

**Comment by darthcloud** [comment]
Is it a DIY build or the Humble dongle?

**Reply by alexgame8818** [reply]
I built a PCE BLUERETRO on PCB which using ESP32 Dev Kit Board on it.
I tried about 50 games,only JackieChan has problem.
(Bomberman is OK after tried to change some controllers. But still dont know why.)

On ESP32 Dev Kit Board,I found the Reset button is also the RUN button for PCE games.
If other people dont have this problem,
MAYBE I should just use an EPS32 chip to build it myself.
(Dev Kit board could let me focus on cables building :)

**Reply by darthcloud** [reply]
The reset button reset the ESP32 so it's normal you will get random button press while its rebooting.
